app/utils/Circe.scala

```diff
@@ -1,11 +1,67 @@
 package utils
 
-import io.circe.Decoder
+import io.circe.{Decoder, Json, JsonObject, Printer}
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+
+import scala.jdk.CollectionConverters._
+
 
 object Circe {
   def decodeStringAndCollect[T](pf: PartialFunction[String,T]): Decoder[T] = Decoder.decodeString.emap { s =>
     pf.lift(s)
       .map(Right.apply)
       .getOrElse(Left(s"Unexpected value: $s"))
   }
+
+  private val printer = Printer.spaces2.copy(dropNullValues = true)
+  def noNulls(json: Json): String = printer.print(json)
+
+  // Converts Circe Json to Dynamodb Attributes
+  def jsonToDynamo(json: Json): AttributeValue =
+    json.deepDropNullValues.fold(
+      jsonNull = AttributeValue.builder().nul(true).build,
```

Comment:
feels like a very amazon-ey move to go for `nul` instead of `null`

Reply:
yes I'm not really a fan of the method names in the sdk!